app/Controllers/userController.php

```diff
@@ -181,8 +181,8 @@ public function manageAction() {
 	}
 
 	public static function createUser($new_user_name, $passwordPlain, $apiPasswordPlain, $userConfig = array(), $insertDefaultFeeds = true) {
-		if (!is_array($userConfig)) {
-			$userConfig = array();
+		if (!is_array($userConfig) || empty($userConfig)) {
+			$userConfig = Minz_Configuration::load(join_path(DATA_PATH, 'config-user.php'));
 		}
```

Comment:
I suggest another way to achieve the same thing while avoiding changing the calls of `createUser`:

- don't change the condition above
- but rename actual `$userConfig` in `$userConfigOverride`
- then, add these lines:

```php
$userConfig = Minz_Configuration::load(join_path(DATA_PATH, 'config-user.php'));
$userConfig = array_merge($userConfig, $userConfigOverride);
```